pontoon/base/static/js/table.js

```diff
@@ -9,18 +9,22 @@ $.expr[':'].containsi = function (a, i, m) {
 };
 
 /* Latest activity tooltip */
-var date_formatter = new Intl.DateTimeFormat('en-GB', {
```

Comment:
I'd just replace `en-GB` with `en-US` here and leave everything else as it is on master.

I'd even keep `month: 'long'`, because the space is not limited. But to be consistent, we then also need to set `month` value to `long` in `shortDateFormat` in pontoon/contributors/static/js/profile.js.